migrations/20190717095753_alter_messages_unique_slack_event_id.js

```diff
@@ -0,0 +1,27 @@
+exports.up = function (knex, Promise) {
+  return Promise.all([
+    knex('points')
+      .whereNotIn('message_id',
+        knex('messages')
+          .groupBy('slack_event_id')
+          .min('id')
+      ).del(),
+    knex('messages')
+      .whereNotIn('id',
+        knex('messages')
+          .groupBy('slack_event_id')
+          .min('id')
+      ).del(),
+    knex.schema.alterTable('messages', table => {
+      table.unique('slack_event_id');
+    }),
+  ]);
+};
```

Comment:
Do you know if these execute in a transaction? Because `Promise.all` will fail if any of the promises fail.
If they are not in a transaction then you could have half-migrated code where the first promise completes and the second rejects.

I'm pretty sure `Promise.all` waits for all promises to settle before it resolves/rejects itself.